index.bs

```diff
@@ -690,8 +690,7 @@ user agents must run the following steps:
 
 
 2. Let |n| be a new string of the same size as the {{fileName}} argument to the constructor.
-  Copy every character from {{fileName}} to |n|,
-  replacing any "/" character (U+002F SOLIDUS) with a ":" (U+003A COLON).
+  Copy every character from {{fileName}} to |n|.
```

Comment:
We could make simplify this to

> Let |n| be the {{fileName}} argument to the constructor.

(There are further simplifications possible, but that would require more refactoring than warranted here.)

Reply:
Done